packages/graph/deployment/docker-compose.yml

```diff
@@ -1,7 +1,8 @@
 version: "3.9"
+
 volumes:
   hash-graph-data:
-    external: true
```

Comment:
Does making docker-compose take care of this volume mean it gets destroyed when we "down" the deployment?

Reply:
No, it will only create the volume if it does not exist. It's probably called `deployment_hash-graph-data` (based on the directory it's in).